### DISCUSSION TITLE
Print invoice using bluetooth printer

### DISCUSSION THREAD

**Discussion by prabodhana** [ionic-team/capacitor#4818] [Q&A]
Hello,
I'm start to develop mobile application using vue and capacitor to print invoice. For that I'm using dot matrix printer with external Bluetooth adapter. Can any one guide me to what need I do for connect that Bluetooth adapter with my mobile app.

Thank you.

**Comment by vitalijalbu** [comment]
me too, i'm looking for this type of plugin
